docs/docs/standard_library/logging.md

```diff
@@ -17,14 +18,13 @@ keywords:
   ]
 ---
 
-The standard library provides a familiar `println` statement you can use. Despite being a limited
-implementation of rust's `println!` macro, this construct can be useful for debugging.
+The standard library provides two familiar statements you can use: `println` and `print`. Despite being a limited implementation of rust's `println!` and `print!` macros, these constructs can be useful for debugging.
 
-You can print the output of println statements in your Noir code by using the `nargo execute` command or the `--show-output` flag when using `nargo test` (provided there are println statements in your tests).
+You can print the output of both statements in your Noir code by using the `nargo execute` command or the `--show-output` flag when using `nargo test` (provided there are print statements in your tests).
 
-It is recommended to use `nargo execute` if you want to debug failing constrains with `println` statements. This is due to every input in a test being a constant rather than a witness, so we issue an error during compilation while we only print during execution (which comes after compilation). `println` will not work for failed constraints caught at compile time.
+It is recommended to use `nargo execute` if you want to debug failing constrains with `println` or `print` statements. This is due to every input in a test being a constant rather than a witness, so we issue an error during compilation while we only print during execution (which comes after compilation). Neither `println`, nor `print` are callable for failed constraints caught at compile time.
 
-The `println` statement is unconstrained, so it works for outputting integers, fields, strings, and even structs or expressions. For example:
+Both `print` and `println` statements have the `unconstrained` attribute, enabling them to output integers, fields, strings, and even structs or expressions. For example:
```

Comment:
```suggestion
Both `print` and `println` are generic functions which can work on integers, fields, strings, and even structs or expressions. Note however, that slices are currently unsupported. For example:
```

Reply:
Not sure how this blurb originally got into the docs but `unconstrained` doesn't relate to the function's ability to accept different parameter types at all.